mindmeld/auto_annotator.py

```diff
@@ -764,33 +804,59 @@ def _resolve_person(self, entity):
         Returns:
             entity (dict): A resolved entity dict.
         """
-        entity["dim"] = self.ANNOTATOR_TO_DUCKLING_ENTITY_MAPPINGS[entity["dim"]]
+        entity["dim"] = ANNOTATOR_TO_SYS_ENTITY_MAPPINGS[entity["dim"]]
 
-        if len(entity["body"]) >= 2 and entity["body"][-2:] == "'s":
+        if (
+            self.language == ENGLISH_LANGUAGE_CODE
+            and len(entity["body"]) >= 2
+            and entity["body"][-2:] == "'s"
```

Comment:
Nit: This conditional can be packaged in a boolean function with a descriptive name like `is_plural_entity` or something more specific.

Reply:
Done